src/XRP/issued-currency-client.ts

```diff
@@ -81,8 +85,16 @@ export default class IssuedCurrencyClient {
     )
 
     if (accountLinesResponse.result.error) {
-      throw XrpError.accountNotFound
+      if (accountLinesResponse.result.error === 'actNotFound') {
```

Comment:
good idea - it's actually used in two places, and there's also the `invalidHotwallet` that you note below.  I made a class for these with static attributes to represent the error messages.  To answer your question below - these aren't exactly standard... there are a few error messages that apparently are standard, and then a list of what the various messages might be documented for each API, and it seems there's a lot of consistency.  I think it's reasonable to just add these as constants on a case-by-case basis... this might be the last time we implement something like this for JSON RPC at least, but we might have to revisit this if we do a big refactor off of gRPC